docs/tutorial/upgrading-kubernetes.md

```diff
@@ -2,16 +2,19 @@
 
 Upgrading kubernetes is very easy with kops, as long as you are using a compatible version of kops.
 The kops `1.8.x` series (for example) supports the kubernetes 1.6, 1.7 and 1.8 series,
-as per the kubernetes deprecation policy.  Older versions of kubernetes will likely still work, but these
-are on a best-effort basis and will have little if any testing.  kops `1.8` will not support the kubernetes
-`1.9` series, and for full support of kubernetes `1.9` it is best to wait for the kops `1.9` series release.
+as per the kubernetes deprecation policy.
+
+Older versions of kubernetes will likely still work, but these
+are on a best-effort basis and will have little if any testing.  kops `1.9` will not support the kubernetes
```

Comment:
Since kops 1.9 has been released, let's do it in the present. ie: kops 1.9 does not support kubernetes 1.10.